boards/mcb2388/include/periph_conf.h

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) 2019 Beuth Hochschule für Technik Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_mcb2388
+ * @{
+ *
+ * @file
+ * @brief       MCB2388 peripheral configuration
+ *
+ * @author      Benjamin Valentin <benpicco@beuth-hochschule.de>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Clock configuration
+ * @{
+ */
+#define XTAL_HZ             (12000000U)         /* the board provides a 12 MHz XTAL */
+
+#define CLOCK_CORECLOCK     (72000000U)         /* the lpc2388 runs with 72MHz */
+
+#define CLOCK_PCLK          (CLOCK_CORECLOCK)
+/** @} */
+
+/**
+ * @name    Configuration for the 16x2 character LCD display
+ * @{
+ */
+#define HD44780_PARAMS {            \
```

Comment:
The usual way is to define each param macros that needs to be overridden instead of the whole struct one.

Reply:
And these defines should go in board.h